example/http/server/stackless/http_server_stackless.cpp

```diff
@@ -97,18 +97,15 @@ path_cat(
     return result;
 }
 
-// This function produces an HTTP response for the given
-// request. The type of the response object depends on the
-// contents of the request, so the interface requires the
-// caller to pass a generic lambda for receiving the response.
-template<
-    class Body, class Allocator,
-    class Send>
-void
+// This function produces an HTTP response for the given request.
+// The concrete type of the response message (which depends on
+// the request), is type-erased in the message_generator return
+// value.
+template <class Body, class Allocator>
+http::message_generator
 handle_request(
```

Comment:
Are all the `handle_request` functions in the examples still exactly the same?

Reply:
Didn't check, but fully expect it indeed. The reason I didn't check is because this realization seemed so blindingly obvious, that I was 1000% certain that this was on purpose and the priority was to make each flavour of server examples self-contained¹.

However, now that type-erasure is in the mix, there's incentive to put it separately, as it might make a considerable dent in build times with a suitable `libbeast-http-server-logic` static target, or so.

¹(module certificate business)

Reply:
Re-reading, you likely weren't talking about extracting a translation unit - Yes I applied only identical transformations to identical functions in various examples.